src/Core/Geographic/Extent.js

```diff
@@ -232,22 +232,23 @@ Extent.prototype.dimensions = function dimensions(unit) {
  * Return true if coord is inside the bounding box.
  *
  * @param {Coordinates} coord
+ * @param {number} delta value to expand extent
  * @return {boolean}
  */
-Extent.prototype.isPointInside = function isPointInside(coord) {
+Extent.prototype.isPointInside = function isPointInside(coord, delta) {
```

Comment:
I'd use `epsilon = 0` rather than `delta`.

And the jsdoc could be replaced with: `coord is inside the extent (+/- epsilon)`.

Reply:
ok